Library/Homebrew/keg_relocate.rb

```diff
@@ -97,7 +99,12 @@ def detect_cxx_stdlibs(_options = {})
   end
 
   def each_unique_file_matching(string)
-    Utils.popen_read("/usr/bin/fgrep", "-lr", string, to_s) do |io|
+    bsd = `/usr/bin/fgrep -V`.include?("BSD grep")
```

Comment:
(or do I want to write e.g. a private each_unique_file_matching_grep_args method for each OS?)